content/docs/2.10/operate/admission-webhooks.md

```diff
@@ -0,0 +1,7 @@
++++
+title = "Admission Webhooks"
+description = "Admission webhooks configurations guidance"
+weight = 100
++++
+
+By default, the admission webhooks are registered with `failurePolicy: Ignore`, this won't block the resources creation/update when the admission controller is not available. To ensure that the validation is always required and perform validation, setting `failurePolicy` to `Fail` is required.
```

Comment:
Just a suggestion to create a header for this section; as we'll add more content in the future and we can deep-link to it

```suggestion
## Validation Enforcement

By default, the admission webhooks are registered with `failurePolicy: Ignore`, this won't block the resources creation/update when the admission controller is not available. To ensure that the validation is always required and perform validation, setting `failurePolicy` to `Fail` is required.
```